Sources/FigmaExportCore/Processor/AssetsProcessor.swift

```diff
@@ -88,6 +88,197 @@ public struct ImagesProcessor: AssetsProcessable {
     }
 }
 
+public extension ColorsProcessor {
+    func process(light: [AssetType], dark: [AssetType]?, lightHC: [AssetType]?, darkHC: [AssetType]?) -> ProcessingPairResult {
```

Comment:
Why haven't you updated the current method `func process(light: [AssetType], dark: [AssetType]?) -> ProcessingPairResult {`.